### PR TITLE
Added project and task name to ClearML UploadConfig

### PR DESCRIPTION
**Pull request by Luke100000**
While not used in the ClearML connector itself, I feel like it belongs here

### CODE REVIEW

File: src/rl_framework/util/saving_and_loading/connector/clearml_connector.py [hunk 1]
```diff
@@ -21,11 +21,15 @@ class ClearMLUploadConfig(UploadConfig):
...
     video_length (int): Length of video in frames (which should be generated and uploaded to the connector).
         No video is uploaded if length is 0 or negative.
+    project_name (str): Project path in which the experiment run(s) will be created
+    task_name (Optional str): Name of experiment, None to use the script name
     """
 
     file_name: str
     n_eval_episodes: int
     video_length: int
+    project_name: str
```
Comment (alexander-zap) [inline src/rl_framework/util/saving_and_loading/connector/clearml_connector.py:31]: Should we make this optional as well to not break the existing interface?

Comment (Luke100000) [reply src/rl_framework/util/saving_and_loading/connector/clearml_connector.py:31]: I actually removed the default again, but a generic one like "default" probably would not hurt. On the other side, ClearML itself does not define a default. So every implementation HAS to define their own default or fallback.

**Comment by alexander-zap** [comment]
ClearML will from now on only be using the provided task for logging and no other tasks should be created in parallel. Do we in this case still require these attributes in the ClearML upload? Otherwise we can cancel this PR.

**Comment by Luke100000** [comment]
If the connector itself never initializes a task, no.

**Pull request closed without merging**